samples/NuGet.Config

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+    <add key="feedz.io/xunit/xunit" value="https://f.feedz.io/xunit/xunit/nuget/index.json" protocolVersion="3" />
+    <add key="local" value="C:\src\visualstudio.xunit\artifacts\packages\" />
```

Comment:
This shouldn't be kept, this is just a temp change for you to test locally (you'll need to update the path as you know).